src/lib/Auth.ts

```diff
@@ -11,6 +10,9 @@ import chalk from 'chalk';
 import { AVAILABLE_ENVS } from './Env';
 import ThemeService from './api/services/theme.service';
 import { getLocalBaseUrl } from '../helper/serve.utils';
+import Debug from './Debug';
+
+const SERVER_TIMER = 60000 * 5;
```

Comment:
@manishprajapati52 keep it to 1 minute only. @manishprajapati52

Reply:
@vivek-gofynd 
One minute would be less if user is not already signed in on partners panel. What do you think?

Reply:
Set 2 minutes